### PR TITLE
Hotfix/qnx localtime

### PR DESCRIPTION
**Pull request by hannemn**
Made QNX buildable with time modules option

### CODE REVIEW

File: contrib/ecaltime/localtime/CMakeLists.txt [hunk 1]
```diff
@@ -45,7 +45,7 @@ target_include_directories(${PROJECT_NAME}
 
 target_link_libraries(${PROJECT_NAME}
   Threads::Threads
-  $<$<AND:$<BOOL:${UNIX}>,$<NOT:$<BOOL:${APPLE}>>>:rt>
+  $<$<AND:$<BOOL:${UNIX}>,$<NOT:$<BOOL:${APPLE}>>,$<NOT:$<BOOL:${QNXNTO}>>>:rt
```
Comment (pablorcum) [inline contrib/ecaltime/localtime/CMakeLists.txt:48]: There's a typo in the escape character `${QNXNTO}>>>:rt` should be `${QNXNTO}>>:rt>`

**Review by pablorcum — CHANGES_REQUESTED** [review]

File: contrib/ecaltime/localtime/CMakeLists.txt (hunk 1 above)
Comment (hannemn) [reply contrib/ecaltime/localtime/CMakeLists.txt]: Should be fixed now :-D

File: contrib/ecaltime/localtime/CMakeLists.txt [hunk 2]
```diff
@@ -45,7 +45,7 @@ target_include_directories(${PROJECT_NAME}
 
 target_link_libraries(${PROJECT_NAME}
   Threads::Threads
-  $<$<AND:$<BOOL:${UNIX}>,$<NOT:$<BOOL:${APPLE}>>>:rt>
+  $<$<AND:$<BOOL:${UNIX}>,$<NOT:$<BOOL:${APPLE}>>,$<NOT:$<BOOL:${QNXNTO}>>:rt>
```
Comment (pablorcum) [inline contrib/ecaltime/localtime/CMakeLists.txt]: I thing there's still a missing >

It should be `${QNXNTO}>>>:rt>`

**Review by pablorcum — CHANGES_REQUESTED** [review]

File: contrib/ecaltime/localtime/CMakeLists.txt (hunk 2 above)
Comment (hannemn) [reply contrib/ecaltime/localtime/CMakeLists.txt]: The latest commit on the PR already contains the version with  `${QNXNTO}>>>:rt>`

**Review by pablorcum — APPROVED** [review]

**Pull request merged**